pkg/workflows/sdk/testutils/runtime.go

```diff
@@ -1,9 +1,19 @@
 package testutils
 
 import (
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk"
 )
 
-type runtime struct{}
+type NopRuntime struct{}
```

Comment:
_nit_ `NoOp` instead of `Nop` as this was initially confusing with `Node Operator`